introspection.go

```diff
@@ -149,6 +149,7 @@ type ConnectionRuntimeState struct {
 	OutboundExchange ExchangeSetRuntimeState `json:"outboundExchange"`
 	Relayer          RelayerRuntimeState     `json:"relayer"`
 	HealthChecks     []bool                  `json:"healthChecks,omitempty"`
+	LastActivity     time.Time               `json:"last_activity"`
```

Comment:
nit: `json:"lastActivity"` (camel case to be consistent with the other fields)

Reply:
Done